ocs_ci/ocs/defaults.py

```diff
@@ -24,6 +24,7 @@
 API_VERSION = "v1"
 CEPHFILESYSTEM_NAME = "ocs-storagecluster-cephfilesystem"
 RBD_PROVISIONER = f"{ROOK_CLUSTER_NAMESPACE}.rbd.csi.ceph.com"
+RBD_NAME = "rbd"
```

Comment:
We should make the RBD poolname configurable via config because it's not constant and can differ in other RHCS clusters.

Reply:
I will make it configurable in next PR